src/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { PublicKey } from '@solana/web3.js';
+
+export function encodeSeeds(seeds: any[]): Buffer {
```

Comment:
`any` is too loose, you're relying on `seed.seed.length` below.

Reply:
Fixed.